benchmark/Dockerfile

```diff
@@ -27,17 +29,18 @@ RUN wget -O - https://packages.adoptium.net/artifactory/api/gpg/key/public | apt
 
 COPY requirements.txt requirements.txt
 RUN python3 -m pip install --upgrade --no-cache-dir --upgrade pip && \
-  python3 -m pip install --upgrade --no-cache-dir setuptools && \
-  python3 -m pip install --upgrade --no-cache-dir -r requirements.txt && \
-  python3 -m pip install --upgrade --no-cache-dir aiomysql && \
-  python3 -m pip check
+    python3 -m pip install --upgrade --no-cache-dir setuptools && \
+    python3 -m pip install --upgrade --no-cache-dir pycairo && \
+    python3 -m pip install --upgrade --no-cache-dir -r requirements.txt && \
+    python3 -m pip install --upgrade --no-cache-dir aiomysql && \
```

Comment:
I know this isn't really to do with your PR, but do you know if there's any reason these all can't be combined into the same pip install? That way any shared transitive dependency versions are resolved only once and we can be fully confident that the pinned versions in the requirements files are actually what gets installed. If I'm not mistaken the separate install for aiomysql could force an upgrade of a previously pinned dependency. I also am confused why aiomysql is even needed in this image.

Reply:
There are a number of things about this I don't understand.
I'm not sure why we need `pycairo` - it's not installed on my system and I don't need it to run benchmarks. Yet without it:
```
42.46 pygobject 3.42.1 requires pycairo, which is not installed.
```
I also don't have pygobject `installed`. If I move this into requirements.txt, I still get this error. I'll keep digging.